ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/IndividualQuestionnaireSteps.cs

```diff
@@ -19,7 +19,12 @@ public sealed class IndividualQuestionnaireSteps : QuestionnaireJourney
         private readonly DecisionJourney _yourInternetConnection;
         private readonly DecisionJourney _accessToRoom;
         private readonly DecisionJourney _consent;
-        private readonly JourneyStepPage _checkYourComputer;
+        private readonly DecisionJourney _checkYourComputer;
```

Comment:
As you mentioned on slack with the duplication, maybe you can move these out to a separate `SelfTestSteps` class?

Reply:
As discussed before, we need to refactor the code in the acceptance tests to reduce duplication.